html/dom/render-blocking/header-inserted-preload-link.tentative.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<title>Parser-inserted preload links with "blocking=render" are render-blocking</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="support/test-render-blocking.js"></script>
+
+<style>
+@font-face {
+  font-family: custom-font;
+  src: url('/fonts/Ahem.ttf?pipe=trickle(d1)');
+}
+</style>
+<span id="target" style="font: 20px/1 custom-font">Lorem ipsum</span>
+
+<script>
+test_render_blocking(
+    null,
```

Comment:
I missed it in the code review...

`test_render_blocking` needs to be able to tell if the render-blocking resource has been loaded. It does so by listening to the `load` event of the element, or if that's impossible (e.g., a script-blocking stylesheet or a parser-blocking script), then it listens to `window.load`.

But here we don't have an element to listen to, and I guess preload link headers do not block `window.load`? Then the test won't work.

Reply:
Hmm. Do you have a suggested better way of testing this? Also curious to get @noamr's thoughts.